libcxx/test/std/time/time.zone/time.zone.timezone/time.zone.members/sys_info.zdump.pass.cpp

```diff
@@ -0,0 +1,130 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17
+// UNSUPPORTED: no-filesystem, no-localization, no-tzdb, has-no-zdump
+
+// XFAIL: libcpp-has-no-incomplete-tzdb
+// XFAIL: availability-tzdb-missing
+
+// TODO TZDB Investigate
+// XFAIL: target={{armv(7|8)l-linux-gnueabihf}}
+
+// This test compares the output of the zdump against the output based on the
```

Comment:
I think this documentation is great. It really improves my ability to quickly comprehend the test. Thank you.

OK, so this is a bit of a ramble, but here's my thinking on where to put the comment. 

The leading comment in the test is large, and my brain largely treats as boiler plate. It's also often sandwiched between that boiler plate, and then the boiler plate of the tests includes and any utility/helper code, 

I've also noticed a lot more comment rot on documentation that goes in the comment header vs documentation placed directly on or in the test function.

Would you be willing to move it down towards the bottom, maybe into or above `main` or whichever function you want to attach it to?

Reply:
As just discussed; from our discussion last week I thought you preferred these on top. I'm totally fine to move them down.

Thanks for the review!
Louis and I will continue our review of this patch next week. I'll address the open issues after that review iteration.